Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -5585,9 +5563,7 @@ void WebPageProxy::didCommitLoadForFrame(FrameIdentifier frameID, FrameInfoData&
         if (is<RemoteLayerTreeDrawingAreaProxy>(*m_drawingArea))
             internals().firstLayerTreeTransactionIdAfterDidCommitLoad = downcast<RemoteLayerTreeDrawingAreaProxy>(*drawingArea()).nextLayerTreeTransactionID();
 #endif
-#if PLATFORM(MAC)
-        m_processActivityState.dropLikelyToUseNotificationsActivity();
-#endif
+        internals().pageAllowedToRunInTheBackgroundToken = nullptr;
```

Comment:
Should we drop this in `didFailLoadForFrame` as well?

Reply:
Yes, we probably should. I'll fix.